test/integration/shared.js

```diff
@@ -38,8 +38,8 @@ function delay(timeout) {
   });
 }
 
-function dropCollection(dbObj, collectionName) {
-  return dbObj.dropCollection(collectionName).catch(ignoreNsNotFound);
+function dropCollection(dbObj, collectionName, options = {}) {
+  return dbObj.dropCollection(collectionName, options).catch(ignoreNsNotFound);
```

Comment:
Just adding the ability to pass options to the `dropCollection` helper. If we pass `encryptedFields` here an extra 3 collections will get dropped. (The esc/ecc/ecoc collections)